code/style.css

```diff
@@ -1,7 +1,286 @@
+}
+
+.hero-image {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.5), rgba(0, 0, 0, 0.5)), url("./hero-image.jpg");
+  height: 50vh;
+  max-height: 300px;
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  position: relative;
+}
+
+.hero-text {
+  text-align: center;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  color: white;
+  font-family: 'Righteous';
+  font-size: 24px;
+}
+
+.slogan {
+  font-size: 18px;
+}
+
+.menu-btn {
+  width: 95%;
+  font-family: 'Righteous';
+  font-size: 28px;
+  border: transparent;
+  font-weight: bold;
+  color: rgb(71, 131, 121);
+  background-color: transparent;
+  padding-top: 10px;
+}
+
+.navbar {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  max-height: 0;
+  overflow: hidden;
+  transition: 0.5s ease-out;
+  width: 90%;
+  border-radius: 10px;
+}
+
+a {
+  text-decoration: none;
+  font-family: 'Righteous';
+  letter-spacing: 0.5mm;
+  font-size: 20px;
+  font-weight: bold;
+  color: rgb(53, 51, 51);
+  padding-top: 10px;
+}
+
+.open+.navbar {
+  max-height: 200px;
+  width: 90%;
+}
+
+.form-faq, .topnav {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.form, .faq {
+  border: rgb(71, 131, 121);
+  background-color: rgb(71, 131, 121);
+  border-style: solid;
+  border-radius: 2%;
+  width: 95%;
+  margin-top: 10px;
+  text-align: center;
+  color: white;
+  box-shadow: 0 5px 5px 0 rgba(0, 0, 0, 0.411);
+  font-family: 'Glory', sans-serif;
+}
+
+.form {
+  padding-bottom: 10px;
+}
+
+.faq {
+  padding-bottom: 30px;
+}
+
+h2 {
+  font-size: 24px;
+  letter-spacing: 0.5mm;
+  padding-bottom: 5px;
+  font-family: 'Righteous';
+}
+
+hr {
+  width: 80%;
+  margin-bottom: 15px;
+  color: rgba(138, 172, 182, 0.849);
+  border-radius: 10px;
+  border: 2px solid;
+}
+
+input {
+  padding: 10px;
+  border: rgb(71, 131, 121) solid 1px;
+  border-radius: 10px;
+  background-color: rgba(138, 172, 182, 0.849);
+  color: white;
+}
 
-.active + .answer {
-  /* some styling here */
+input::placeholder {
+  color: white;
+  font-style: italic;
 }
+
+input:focus {
+  background-color: rgb(90, 111, 129);
+}
+
+.sub-button {
+  transition-duration: 0.4s;
+  border: white solid 1px;
+  background-color: rgb(71, 160, 145);
+  border-radius: 10px;
+  padding: 15px;
+  color: white;
+}
+
+.sub-button:hover {
+  border: rgba(138, 172, 182, 0.849) solid;
+  background-color: rgba(138, 172, 182, 0.849);
+  cursor: pointer;
+  box-shadow: 0 5px 5px 0 rgba(0, 0, 0, 0.411);
+}
+
+label.name, .email, .phonenumber, .terms {
+  display: block;
+  margin: 10px;
+}
+
+.question {
+  color: white;
+  cursor: pointer;
+  padding: 18px;
+  width: 95%;
+  border: rgb(77, 106, 131);
+  text-align: left;
+  outline: none;
+  font-size: 18px;
+  transition: 0.4s;
+  border-radius: 10px;
+  font-family: 'Glory', sans-serif;
+}
+
+.question.one, .three {
```

Comment:
Nice way to do the alternate colours. I think theres an 'even child selector' or something similar that you could use too. The benefit is that you could add 5 or 10 more questions and not have to write anything extra here to make them continue to alternate